test/feature/reverse_routing.feature

```diff
@@ -0,0 +1,31 @@
+Feature: reverse routing
+
+  Background:
+    Given an empty "controllers" directory
```

Comment:
I think your feature specs should all have a basic Express app with Expressive loaded and otherwise empty routes and controllers directories. This should be set up in a global `beforeEach`.